src/boot/boot.c

```diff
@@ -1305,6 +1311,7 @@ static void config_defaults_load_from_file(Config *config, char *content) {
                                 }
                                 config->console_mode = u;
                         }
+
```

Comment:
Please do not insert an unnecessary empty line.

Reply:
updated PR to remove this